tests/raftstore/test_conf_change.rs

```diff
@@ -410,6 +410,9 @@ fn test_split_brain<T: Simulator>(cluster: &mut Cluster<T>) {
     // Disable default max peer number check.
     pd_client.disable_default_rule();
 
+    // Disable safe conf change option to create a split brain.
+    cluster.cfg.raft_store.safe_conf_change = false;
```

Comment:
If the `safe_conf_change` is `true`, it will fail to add a new node (4, 4) here

https://github.com/pingcap/tikv/blob/hhkbp2/safe-conf-change/tests/raftstore/test_conf_change.rs#L429

when there is already one node (1, 1) isolated in the 3 nodes cluster.